apps/web-portal-client/lib/utils/AppPersistentStorage.mjs

```diff
@@ -0,0 +1,67 @@
+const BrowserKeys = {
+  UserToken: 'ZORKO_APP_TOKEN',
+  LastPath: 'ZORKO_APP_LAST_PATH',
+}
+
+class AppPersistentStorage {
```

Comment:
plz, add here a todo with optional tech-debt task, like - refactor to use just getter/setters for manipulation with persistent storage. It would give instead of 6 methods only 4, so much better interface

Reply:
changed to setters/getters

Reply:
cool